crates/lox_core/src/bodies.rs

```diff
@@ -62,11 +62,23 @@ pub fn gravitational_parameter<T: PointMass>(_: T) -> f64 {
     <T as PointMass>::gravitational_parameter()
 }
 
+/// Semantic type alias for f64 in the context of polynomial types and operations.
+pub type PolynomialCoefficient = f64;
+
+/// Right ascension polynomial coefficients.
+///
+/// p2 is implicit, being 0.0 for all supported bodies.
+pub type RACoefficients = [PolynomialCoefficient; 2];
+
+pub trait RotationalElements: Copy {
+    const RIGHT_ASCENSION_COEFFICIENTS: RACoefficients;
```

Comment:
So far, I haven't provided a function accessor to this constant, which would match the interface exposed by other traits, since `SomeBody::RIGHT_ASCENSION_COEFFICIENTS` seems just as convenient as `SomeBody.right_ascension_coefficients`. I may be missing an intended use case though - please let me know.

I don't like to introduce a discrepancy with the other trait implementations though, so we could either implement function accessors for this trait like the others, or consider updating the _other_ traits to use associated consts instead of functions, since these objects are really just sacks of const data. Thoughts?

Reply:
I concur, there is currently no need for an accessor function, since it is my assumption that users will not use the raw coefficients.
The next step after this PR is merged would be the implementation of the actual evaluation of the polynomials which will be the user-facing API.